### ISSUE TITLE
Python 3 support

### ISSUE THREAD

**Issue by eyalev** [quip/quip-api#24]
Does the Python client supports Python 3?

**Comment by dacut** [comment]
No, at least not yet. Attempting `import quip` on Python 3.6 results in:

    In [1]: import quip
      File "/Users/cuthbert/projects/quip-tracker/venv/lib/python3.6/site-packages/quip.py", line 624
        except urllib2.HTTPError, error:
                                ^
    SyntaxError: invalid syntax

I need Python 3 support, so I'll fork, edit, and make a pull request.

**Comment by dacut** [comment]
Pull request created: https://github.com/quip/quip-api/pull/27

**Comment by analyticalmonk** [comment]
When trying to follow the example code from the README using Python 3.6, I got the following error:
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-5-71602af7a755> in <module>()
      1 # help(quip)
      2 client = quip.QuipClient(access_token=u"WWRjQU1BdGxwcVc=|1543052789|xaGfI4hWXPyLEPwsYp/LWhivGpW75YvOegh9jndVHQI=")
----> 3 user = client.get_authenticated_user()

~/sc_dev/quip_experiments/quip.py in get_authenticated_user(self)
    135     def get_authenticated_user(self):
    136         """Returns the user corresponding to our access token."""
--> 137         return self._fetch_json("users/current")
    138 
    139     def get_user(self, id):

~/sc_dev/quip_experiments/quip.py in _fetch_json(self, path, post_data, **args)
    689         try:
    690             return json.loads(
--> 691                 urlopen(request, timeout=self.request_timeout).read())
    692         except HTTPError as error:
    693             try:

/usr/lib/python3.5/json/__init__.py in loads(s, encoding, cls, object_hook, parse_float, parse_int, parse_constant, object_pairs_hook, **kw)
    310     if not isinstance(s, str):
    311         raise TypeError('the JSON object must be str, not {!r}'.format(
--> 312                             s.__class__.__name__))
    313     if s.startswith(u'\ufeff'):
    314         raise JSONDecodeError("Unexpected UTF-8 BOM (decode using utf-8-sig)",

TypeError: the JSON object must be str, not 'bytes'
```

It was a simple fix; opened #29 for the same.
